lib/inky.ex

```diff
@@ -9,21 +9,28 @@ defmodule Inky do
   require Logger
 
   alias Inky.Display
-  alias Inky.RpiHAL
 
   @typedoc "The Inky process name"
   @type name :: atom | {:global, term} | {:via, module, term}
 
   @default_border :black
   @push_timeout 5000
+  @valid_accents [:black, :red, :yellow]
+  @valid_types [
```

Comment:
```suggestion
  @valid_models [
```

Or maybe `variants` if `model` doesn't seem to fit.

Reply:
Perhaps separate those `_with_accents` into another module attribute that you concat with the rest here? Then, you can use that below in the guard that needs them.